src/dmd/mars.d

```diff
@@ -794,6 +694,121 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
     return status;
 }
 
+/**
+ * Parses the command line arguments and configuration files
+ *
+ * Params:
+ *   argc = Number of arguments passed via command line
+ *   argv = Array of string arguments passed via command line
+ *   params = parametes from argv
+ *   files = files from argv
+ *  Returns: true on faiure
```

Comment:
```suggestion
 * Returns: true on faiure
```

Reply:
Usually, the style is to have a newline between the last parameter and the `Returns`.